config/core/test-configs-chromeos.yaml

```diff
@@ -1,32 +1,62 @@
 test_plans:
 
-  cros-boot: &cros-boot
-    pattern: 'chromeos/cros-boot.jinja2'
+  cros-baseline: &cros-baseline
+    pattern: 'chromeos/cros-baseline.jinja2'
     filters: &cros-filters
       - blocklist: { defconfig: ['kselftest'] }
+    params:
+      docker_image: 'kernelci/cros-baseline'
 
-  cros-boot-fixed:
-    <<: *cros-boot
+  cros-baseline-fixed:
+    <<: *cros-baseline
     filters: &cros-filters-fixed
       - blocklist: { defconfig: ['kselftest'] }
       - blocklist: { defconfig: ['cros'] }
     params:
+      docker_image: 'kernelci/cros-baseline'
       fixed_kernel: true
 
-  cros-boot_qemu: &cros-boot_qemu
-    base_name: cros-boot
-    pattern: 'chromeos/cros-boot-qemu.jinja2'
+  cros-baseline_qemu: &cros-baseline_qemu
+    base_name: cros-baseline
+    pattern: 'chromeos/cros-baseline.jinja2'
     filters: &cros-qemu-filters
       - blocklist: { defconfig: ['kselftest'] }
       - passlist: { lab: ['lab-collabora-staging', 'lab-collabora'] }
+    params:
+      base_template: 'chromeos/cros-boot-qemu.jinja2'
+      docker_image: 'kernelci/cros-baseline'
 
-  cros-boot_qemu-fixed:
-    <<: *cros-boot_qemu
-    base_name: cros-boot-fixed
+  cros-baseline_qemu-fixed:
+    <<: *cros-baseline_qemu
+    base_name: cros-baseline-fixed
     filters: &cros-qemu-filters-fixed
       - blocklist: { defconfig: ['kselftest'] }
       - blocklist: { defconfig: ['cros'] }
       - passlist: { lab: ['lab-collabora-staging', 'lab-collabora'] }
+    params:
+      base_template: 'chromeos/cros-boot-qemu.jinja2'
+      docker_image: 'kernelci/cros-baseline'
```

Comment:
This could be inherited from `cros-baseline_qemu` to avoid repeating it.

Reply:
Updated